mail/dovecot/Portfile

```diff
@@ -1,10 +1,12 @@
 PortSystem          1.0
+# This port can be removed on September 8, 2020. 
+PortGroup           obsolete 1.0
```

Comment:
There's usually no point using this portgroup unless you also set replaced_by.

Reply:
OK